Learn-CSS-Blog/index2.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+</body>
+</html>
+
+<!-- 
+    Add 'Basic CSS Blog' between the opening `<title>` and closing `</title>` tags.
```

Comment:
We could instruct learners to write out the whole title here:

Add the text "Home | Learn Basic CSS by building a Blog" between the opening `<title>` and closing `</title>` tags.